profiles/nuvla.json

```diff
@@ -0,0 +1,129 @@
+{
+  "name": "Nuvla",
+  "revision": "2015-10-13",
+  "vendor_verified": "2015-10-13",
+  "url": "http://nuv.la/login",
+  "status": "production",
+  "status_since": "2012-01-01",
+  "type": "Generic",
```

Comment:
Your doing mainly IaaS management as far as I can see so `IaaS-centric` is more appropriate here I think. See http://www.uni-bamberg.de/?id=85921 for more information.